components/mpas-ocean/src/mode_init/mpas_ocn_init_vertical_grids.F

```diff
@@ -163,66 +164,66 @@ subroutine ocn_generate_60layerPHC_vertical_grid(interfaceLocations)!{{{
-      interfaceLocations(2) = 500_RKIND
-      interfaceLocations(3) = 1500_RKIND
-      interfaceLocations(4) = 2500_RKIND
-      interfaceLocations(5) = 3500_RKIND
-      interfaceLocations(6) = 4500_RKIND
-      interfaceLocations(7) = 5500_RKIND
-      interfaceLocations(8) = 6500_RKIND
-      interfaceLocations(9) = 7500_RKIND
-      interfaceLocations(10) = 8500_RKIND
-      interfaceLocations(11) = 9500_RKIND
-      interfaceLocations(12) = 10500_RKIND
-      interfaceLocations(13) = 11500_RKIND
-      interfaceLocations(14) = 12500_RKIND
-      interfaceLocations(15) = 13500_RKIND
-      interfaceLocations(16) = 14500_RKIND
-      interfaceLocations(17) = 15500_RKIND
-      interfaceLocations(18) = 16509.83984375_RKIND
-      interfaceLocations(19) = 17547.904296875_RKIND
-      interfaceLocations(20) = 18629.125_RKIND
-      interfaceLocations(21) = 19766.025390625_RKIND
-      interfaceLocations(22) = 20971.134765625_RKIND
-      interfaceLocations(23) = 22257.826171875_RKIND
-      interfaceLocations(24) = 23640.880859375_RKIND
-      interfaceLocations(25) = 25137.013671875_RKIND
-      interfaceLocations(26) = 26765.416015625_RKIND
-      interfaceLocations(27) = 28548.361328125_RKIND
-      interfaceLocations(28) = 30511.91796875_RKIND
-      interfaceLocations(29) = 32686.794921875_RKIND
-      interfaceLocations(30) = 35109.34375_RKIND
-      interfaceLocations(31) = 37822.75390625_RKIND
-      interfaceLocations(32) = 40878.4609375_RKIND
-      interfaceLocations(33) = 44337.765625_RKIND
-      interfaceLocations(34) = 48273.66796875_RKIND
-      interfaceLocations(35) = 52772.796875_RKIND
-      interfaceLocations(36) = 57937.28515625_RKIND
-      interfaceLocations(37) = 63886.2578125_RKIND
-      interfaceLocations(38) = 70756.328125_RKIND
-      interfaceLocations(39) = 78700.25_RKIND
-      interfaceLocations(40) = 87882.5234375_RKIND
-      interfaceLocations(41) = 98470.5859375_RKIND
-      interfaceLocations(42) = 110620.421875_RKIND
-      interfaceLocations(43) = 124456.6953125_RKIND
-      interfaceLocations(44) = 140049.71875_RKIND
-      interfaceLocations(45) = 157394.640625_RKIND
-      interfaceLocations(46) = 176400.328125_RKIND
-      interfaceLocations(47) = 196894.421875_RKIND
-      interfaceLocations(48) = 218645.65625_RKIND
-      interfaceLocations(49) = 241397.15625_RKIND
-      interfaceLocations(50) = 264900.125_RKIND
-      interfaceLocations(51) = 288938.46875_RKIND
-      interfaceLocations(52) = 313340.46875_RKIND
-      interfaceLocations(53) = 337979.375_RKIND
-      interfaceLocations(54) = 362767.0625_RKIND
-      interfaceLocations(55) = 387645.21875_RKIND
-      interfaceLocations(56) = 412576.84375_RKIND
-      interfaceLocations(57) = 437539.28125_RKIND
-      interfaceLocations(58) = 462519.0625_RKIND
-      interfaceLocations(59) = 487508.375_RKIND
-      interfaceLocations(60) = 512502.84375_RKIND
-      interfaceLocations(61) = 537500_RKIND
+      interfaceLocations(2) = 10.0_RKIND
```

Comment:
Note: I have switched from cm to m here, but the `interfaceLocations` get normalized to be between 0.0 and 1.0 below, so the units don't matter.